guides/runtime-parser.md

```diff
@@ -0,0 +1,323 @@
+# Introduction
+
+*This guide assumes that the [App Shell guide](https://mobile.angular.io/guides/app-shell.html) has been completed.*
+
+Now we're going to take a look at another part of the Angular Mobile Toolkit - the App Shell Runtime Parser. The Runtime Parser is a library which has the following features:
+
+- Automatically generates the App Shell of our application by using a predefined template.
+- Provides the App Shell template from the local cache automatically for each view that should use it.
+
+This way, when the user navigates to a different view in the application she will instantly get a minimal working UI that will be later replaced with the view's full functionality. You can find more about the Application Shell concept in the context of [Progressive Web Applications](https://developers.google.com/web/progressive-web-apps/) (PWA) on the [following link](https://developers.google.com/web/updates/2015/11/app-shell?hl=en).
```

Comment:
The terms "App Shell" and "Application Shell" are used interchangeably, which is a little distracting. I think on of the two should be used consistently (or at least it should be mentioned that the terms are synonyms).